lib/base-compiler.ts

```diff
@@ -1599,7 +1599,7 @@ export class BaseCompiler implements ICompiler {
         return [{text: 'Internal error; unable to open output path'}];
     }
 
-    getIrOutputFilename(inputFilename: string, filters: ParseFiltersAndOutputOptions): string {
+    getIrOutputFilename(inputFilename: string, filters?: ParseFiltersAndOutputOptions): string {
```

Comment:
Explanation of this change: we want to call this function inside `runCompiler`, but `filters` are optional there: https://github.com/compiler-explorer/compiler-explorer/blob/8dd09d1c1f0e44db48d6fa42083a42377dd572fe/lib/base-compiler.ts#L482

Reply:
Thanks for the explanation :+1: